docs/releasing.md

```diff
@@ -3,6 +3,10 @@
 <!-- SPDX-License-Identifier: CC0-1.0 -->
 <!-- SPDX-FileCopyrightText: 2021-2024 The Foundation for Public Code <info@publiccode.net>, https://standard.publiccode.net/AUTHORS -->
 
+Releases are created by automation when triggered by tags pushed to the repository.
+When the release branch is expected to be good enough to create a release, the maintainers first create and push a release candidate tag.
+If the release artifacts created by the automation look good, then the maintainers create and push a release tag.
```

Comment:
I think within the intro paragraph, it would be good to frame the purpose of the document, and also briefly mention which tasks are part of the procedure, and ultimately the expected end result.

Here's a suggestion:

```suggestion
This step-by-step guide walks you through the process of preparing a release for Standard for Public Code, from reviewing the current state of the `develop` branch to sending the final files for print. It covers everything you need to ensure the release is properly updated, reviewed, and tagged, including updating relevant documentation, conducting reviews, and publishing the release. Additionally, it includes final tasks like preparing print materials and notifying translation contributors.
```

Reply:
Got it! I've adjust the suggestion accordingly. Yes, please amend the suggestion if you have further changes.